client/src/components/ExpandableSection.vue

```diff
@@ -36,11 +37,7 @@ function toggle() {
       <slot name="expanded"></slot>
     </div>
 
-    <button
-      class="ExpandableSection__Expander"
-      @click="toggle"
-      :aria-label="_label"
-    >
+    <button class="ExpandableSection__Expander" :aria-label="_label">
```

Comment:
Since this no longer has the click event, what happens when I click it? The video in the PR only shows clicking the general card, not this button.

Reply:
The event bubbles up, so it still works as intended